javascript/README.md

```diff
@@ -17,6 +17,7 @@ to parse individual binary messages.packets.
 
 ### Requirements
 ```
+pip install tox
```

Comment:
@jkretzmer yo, what's tox being used for here? Isn't that only used in the Python bindings?

Reply:
from the top level Makefile

the javascript target is:
`javascript: deps-javascript gen-javascript test-javascript`

gen-javascript is:
`gen-javascript:
	$(call announce-begin,"Generating JavaScript bindings")
	cd $(SWIFTNAV_ROOT)/generator; \
	$(SBP_GEN_BIN) -i $(SBP_SPEC_DIR) \
		       -o $(SWIFTNAV_ROOT)/javascript/sbp/ \
                       -r $(SBP_MAJOR_VERSION).$(SBP_MINOR_VERSION) \
		       --javascript
	cd $(SWIFTNAV_ROOT)
	$(call announce-begin,"Bumping NPM version")
	@- npm version "v$(SBP_MAJOR_VERSION).$(SBP_MINOR_VERSION).$(SBP_PATCH_VERSION)" --no-git-tag-version >/dev/null 2>&1
	$(call announce-end,"Finished generating JavaScript bindings. Please check $(SWIFTNAV_ROOT)/javascript/sbp")`

The SBP_GEN_BIN is tox. `make javascript` fails without it being installed.

Reply:
ooof